src/index.html

```diff
@@ -7,8 +7,46 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="icon">
+        <img src="images/logo.png" alt="">
+      </a>
+      <nav class="nav">
+        <ul class="list">
+          <li class="item">
+            <a href="#" class="link is-active">APPLE</a>
```

Comment:
Add mock hrefs to all links
```suggestion
            <a href="#apple" class="link is-active">APPLE</a>
```